src/sentry/grouping/ingest/seer.py

```diff
@@ -541,9 +550,62 @@ def maybe_check_seer_for_matching_grouphash(
                 date_added=gh_metadata.date_added or timestamp,
                 seer_date_sent=gh_metadata.date_added or timestamp,
                 seer_event_sent=event.event_id,
-                seer_model=SEER_SIMILARITY_MODEL_VERSION,
+                seer_model=model_version.value,
                 seer_matched_grouphash=seer_matched_grouphash,
                 seer_match_distance=seer_match_distance,
             )
 
     return seer_matched_grouphash
+
+
+@sentry_sdk.tracing.trace
+def maybe_send_seer_for_new_model_training(
+    event: Event,
+    existing_grouphash: GroupHash,
+    variants: dict[str, BaseVariant],
+) -> None:
+    """
+    Send a training_mode=true request to Seer to build embeddings for the new model
+    version if the existing grouphash hasn't been sent to the new version yet.
+
+    This only happens for projects that have the new model rolled out. It helps
+    build embeddings for existing groups without affecting production grouping decisions.
+
+    Args:
+        event: The event being grouped
+        existing_grouphash: The grouphash that was found for this event
+        variants: Grouping variants for the event
+    """
+
+    # Check if we should send embeddings for the new model
+    gh_metadata = existing_grouphash.metadata
+    grouphash_seer_model = gh_metadata.seer_model if gh_metadata else None
+
+    if should_send_new_model_embeddings(event.project, grouphash_seer_model):
+        had_metadata = gh_metadata is not None
+        # Send training mode request (honor all checks like rate limits, circuit breaker, etc.)
+        if should_call_seer_for_grouping(event, variants, existing_grouphash):
+            record_did_call_seer_metric(event, call_made=True, blocker="none")
+
+            try:
+                # Call Seer with training_mode=True (results won't be used for grouping)
+                get_seer_similar_issues(event, existing_grouphash, variants, training_mode=True)
+
+                # Record metrics for new model embedding requests
+                metrics.incr(
```

Comment:
seems like we have a bunch of telemetry in `get_seer_similar_issues` already, should we just consolidate there with a `training_mode` tag?